frontend/packages/ux-editor-v3/src/hooks/queries/useFormLayoutsQuery.ts

```diff
@@ -23,5 +23,6 @@ export const useFormLayoutsQuery = (
         dispatch(FormLayoutActions.setInvalidLayouts(invalidLayouts));
         return convertedLayouts;
       }),
+    staleTime: Infinity,
```

Comment:
I am not so familiar with this prop in useQuery, but does it mean that it will never fetch new data and always fetch from cache? 🧐

Reply:
It will always fetch from the cache as long as it has a valid cache, which in this case is infinite (always), but it will perform a new fetch every time we clear the cache. 😊

```
 await queryClient.invalidateQueries({
     queryKey: [QueryKey.FormLayouts, org, app, layoutSetName],
 });
```

![image](https://github.com/Altinn/altinn-studio/assets/46874830/edeac5d4-0dd7-4933-bc61-a4df533d8bf3)

Reply:
The existing code also directly adds things to the cache instead of invalidating it. Both approaches will work to display new data to the user. 😊